ui-tests/cypress/support/pageObjects/administration/emailConfiguration/EmailConfigurationPage.js

```diff
@@ -89,6 +84,80 @@ export default class EmailConfigurationPage extends Content {
         return this.getToolButton()
                     .find(`${htmlElements.button}[type=submit]`);
     }
+     //Sender Functions
+    getSenderManagementTab(){
+        return this.getContents()
+                    .find(this.emailConfigurationTabList)
+                    .children()
+                    .eq(0);
+    }
+    
+    openSender(){
+        return this.getSenderManagementTab().click();
+    }
+
+    getSenderMngt(){
+        return this.getContents()
+                   .find(`${htmlElements.div}.EmailConfigSenderMgmt`);
+    }
+
+    getSenderTable(){
+        return this.getSenderMngt()
+                   .find(`${htmlElements.table}.table`);
+    }
+
+    getTabHeaders(){
+        return this.getSenderTable()
+                    .children(htmlElements.thead)
+                    .children(htmlElements.tr);
+    }
+
+    getAddButton(){
+        return this.getSenderMngt()
+                   .children(htmlElements.a);
+    }
+
+    getSenderForm(){
+        return this.getContents()
+                    .find(`${htmlElements.form}.form-horizontal`);
+    }
+
+    getCodeInput(){
+        return this.getSenderForm()
+                    .find(`${htmlElements.div}.form-group`).eq(0)
+                    .find(`${htmlElements.input}[name="code"]`);
+    }
+
+    getEmailInput(){
+        return this.getSenderForm()
+                    .find(`${htmlElements.div}.form-group`).eq(1)
+                    .find(`${htmlElements.input}[name="email"]`);                  
+    }
+
+    getActionButton(){
```

Comment:
which action button? is it going to always use the first action button available?
Please, make it more generic (and reliable)